src/main/kotlin/no/nav/syfo/access/domain/UserAccessStatus.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.syfo.access.domain
+
+data class UserAccessStatus(
+    var fnr: String?,
+    val canUserBeDigitallyNotified: Boolean,
+    val canUserBePhysicallyNotified: Boolean,
+    val isKode6Eller7: Boolean?, // har adressebeskyttelse
+    val isKanVarsles: Boolean?, // status i KRR: [reservert/ikke reservert + kontakt info nyere enn 18mnd]
```

Comment:
Kanskje litt mer konsistent navngivning mtp engelsk og norsk (isKanVarsles, canUserBeXXNotified), samt hvordan navngi booleans (canUser, isKan)

Reply:
Vi hadde vel en prat om at for domene-termer så kan og bør vi bruke norsk